demo/text-area-demos.html

```diff
@@ -48,6 +48,18 @@ <h3>Minimum Height</h3>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Display the Clear Button</h3>
```

Comment:
I’d think this is a redundant example, as the top of the page says that text area has all the same features as text field. For example, we are not duplicating examples for label, required, or any other attribute.

Same thing for password field examples.

Reply:
> Same thing for password field examples.

Agree about the textarea, but I was personally interested in the password field example, particularly how the clear button and the reveal-password button look and work together.

Reply:
Removed from password-field and text-area demos, but can restore at any point of time